hws/src/dr_ste.py

```diff
@@ -264,9 +264,20 @@ def dump_actions(self, verbosity, tabs):
                 _str += _tabs + action
                 flag = True
 
-        obj = _db._term_dest_db.get(hex(self.hit_addr))
-        if obj != None:
-            _str += _tabs + obj.get("type") + ': ' + obj.get("id") + '\n'
```

Comment:
you can keep this and just add this:
_str += _tabs + obj.get("type") + ': ' + obj.get("id") + ' (addr:  ' +  hex(self.hit_addr) + ')\n'
in the last STE always the next table TIR or VPORT will be printed, just in case of DROP we did not output, and this can be handled if needed

Reply:
Originally I had some simpler code that always printed the additional address information, but you objected to the dump becoming more verbose. The latest version of the code only adds the additional information in specific cases, such as last STE in a chain, or verbose dumps, printing the destination GVMI ID only when it's non-default i.e. not the current GVMI, etc. I'm happy to make every STE print the full GVMI:STE_index information if you want.